.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
+    "tabWidth": 4,
+    "printWidth": 110
```

Comment:
Feel free to adjust these to your desire. https://prettier.io/docs/en/options.html